sdks/python/apache_beam/yaml/yaml_mapping.py

```diff
@@ -616,6 +617,13 @@ def _PyJsFilter(
 
   See more complete documentation on
   [YAML Filtering](https://beam.apache.org/documentation/sdks/yaml-udf/#filtering).
+
+  Args:
+    keep: An expression evaluating to true for those records that should be kept.
+    language: The language of the above expression.
+      Defaults to generic.
+    error_handling: (Optional) Whether and where to output records that throw errors when
+      the above expressions are evaluated.
```

Comment:
#33060 will make it so that all instances of error_handling are treated as Optional (so in this case, it would be printed twice)
```suggestion
    error_handling: Whether and where to output records that throw errors when
      the above expressions are evaluated.
```

Reply:
Good point. Rebased and resolved.

Reply:
Looks like it is still there

Reply:
Fixed.